### ISSUE TITLE
Conceptual question!

### ISSUE THREAD

**Issue by ahmadrv** [google-research/leaf-audio#10]
Hi there,

I'm novice in data science and trying to use the LEAF for some educational tasks. I've confused about the head layer. In my task, i should classify some audios in 11 classes. After training phase, when i try to predict some sample of data, because of head layer that is like:

`num_outputs = 11`
`self._head = tf.keras.layers.Dense(num_outputs, activation=None)`

it returns an 11-dim vector that shows which class the sample belongs. Each class(dim) have maximum value that is the answer but it doesn't work for me! for every sample, the 10th element in output vector have maximum value!

Is there someone show me the problem?
